### PR TITLE
feat: add documentation for openFilesDialog execute method

### PR DESCRIPTION
**Pull request by vladmonday**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -316,6 +316,31 @@ monday.execute("notice", {
 });
 ```
 
+#### Open files preview dialog
+Opens a popup card with the preview of an asset
+
+**type**
+`'openFilesDialog'`
```
Comment (etaylib) [inline README.md:323]: image

we currently support this 
this means you'll get the viewer with all of the files in the PULSE

app should live on top of board/view/item (of the files passed) - meaning have the boardEntities of the board and its assets)

if you want to show only a single image/pass a list of images regardless of their pulse (but same board), we just need to change the context upon the logic in files-preview-logic-connector.js (or add new logic) - very easy

File: README.md [hunk 2]
```diff
@@ -316,6 +316,31 @@ monday.execute("notice", {
...
+
+**params**
+
+| Parameter|Type | Description | Required | Default Value |
+| --- | --- | --- | --- | --- |
+| boardId| Integer | The ID of the board | Yes | |
+| itemId| Integer | The ID of the item, which contains an asset | Yes | |
```
Comment (etaylib) [inline README.md:330]: item id and column id can be mandatory (see files-preview-logic-connector.js) -  if you find it hard for them to pass it

File: README.md [hunk 3]
```diff
@@ -316,6 +316,31 @@ monday.execute("notice", {
...
+| columnId| String | The ID of the column, which contains an asset | Yes | |
+| assetId| Integer | The ID of the asset to open | Yes | |
+
+**Example**
+```javascript
+monday.execute('openFilesDialog', {
```
Comment (etaylib) [inline README.md:336]: we get assetIds from the params - is it generic or only docs use case?

image

**Review by etaylib — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (vladmonday) [reply README.md:336]: I don't know, but it doesn't work for me on prod + I think it's not very clear UI and I don't see much use cases for it

File: README.md [hunk 4]
```diff
@@ -316,6 +316,31 @@ monday.execute("notice", {
 });
 ```
 
+#### Open files preview dialog
+Opens a popup card with the preview of an asset
```
[the remark this answers is not in the document]
Comment (vladmonday) [reply README.md]: fixed

**Pull request merged**